### PR TITLE
Lowers the Cargo Express console's cost multiplier

### PR DESCRIPTION
**Pull request by MrDoomBringer**
Price multiplier of the cargo express console changed to 1x, down from 2x

:cl: MrDoomBringer
tweak: Advances in Rapid Delivery Technology have yielded a reduced premium on express cargo orders! Orders from the express console now have a cost multiplier of 1, down from 2.
/:cl:

[why]: the express console's price was arbitrarily balanced during a time when cargo could get 100K points within a few minutes of the round. As such, the express console's cost multiplier has gone down to something sensible that wont completely discourage its use.

**edit: cost multiplier is now 1x waddyall think**
**edit2boogaloo: emag cost was lowered as a result. keeps the same ratio**

### CODE REVIEW

File: code/modules/cargo/expressconsole.dm [hunk 1]
```diff
@@ -55,9 +55,9 @@
 			continue// by using someone else's crate
 		if(!(obj_flags & EMAGGED) && P.contraband) // will you show me?
 			continue // i'd be right happy to
-		meme_pack_data[P.group]["packs"] += list(list(
+		meme_pack_data[P.group]["packs"] += list(list(//mfw goof sullies your nice cargo features with n a s t y and downright s i l l y comments^^^
 			"name" = P.name,
-			"cost" = P.cost * 2, //displays twice the normal cost
+			"cost" = P.cost * 1.2, //displays twice the normal cost
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/expressconsole.dm]: comment

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

File: code/modules/cargo/expressconsole.dm (hunk 1 above)
Comment (MrDoomBringer) [reply code/modules/cargo/expressconsole.dm]: whuip

File: code/modules/cargo/expressconsole.dm [hunk 2]
```diff
@@ -55,9 +55,9 @@
 			continue// by using someone else's crate
 		if(!(obj_flags & EMAGGED) && P.contraband) // will you show me?
 			continue // i'd be right happy to
-		meme_pack_data[P.group]["packs"] += list(list(
+		meme_pack_data[P.group]["packs"] += list(list(//mfw goof sullies your nice cargo features with n a s t y and downright s i l l y comments^^^
 			"name" = P.name,
-			"cost" = P.cost * 2, //displays twice the normal cost
+			"cost" = P.cost * 1.2, //displays 0.2x normal cost
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/cargo/expressconsole.dm]: um it's not marked down 80%.

Just remove the comment or replace it with the rationale for having it a slightly higher cost so people don't try to make a "consistency" pr in the future.

**Comment by ExcessiveUseOfCobblestone** [comment]
I have no idea if this is fair or not cuz I just sit and watch stocks but the rationale seems sane :^)

**Comment by Buggy123** [comment]
tbh, its cost multiplier should just be 1.

The tradeoff for speedy delivery is the fact that you _blast fucking holes in your department_.

**Comment by MrDoomBringer** [comment]
@Buggy123 i would, but you can upgrade it to remove that issue. I dunno, maybe...

File: code/modules/cargo/expressconsole.dm (hunk 2 above)
Comment (MrDoomBringer) [reply code/modules/cargo/expressconsole.dm]: mfw im retard

**Comment by AlexTheSysop42** [comment]
even the bluespace pods leave a blast iirc

**Comment by MrDoomBringer** [comment]
alright i've lowered it down to 1x because tbh techwebsgating and explosiongating and botheringtomaketheconsolegating are enough imo

File: code/modules/cargo/expressconsole.dm [hunk 3]
```diff
@@ -55,9 +55,9 @@
 			continue// by using someone else's crate
 		if(!(obj_flags & EMAGGED) && P.contraband) // will you show me?
 			continue // i'd be right happy to
-		meme_pack_data[P.group]["packs"] += list(list(
+		meme_pack_data[P.group]["packs"] += list(list(//mfw goof sullies your nice cargo features with n a s t y and downright s i l l y comments^^^
```
Comment (Jordie0608) [inline code/modules/cargo/expressconsole.dm]: Remove comment, `R` `e` `m` `o` `v` `e`

File: code/modules/cargo/expressconsole.dm [hunk 4]
```diff
@@ -55,9 +55,9 @@
 			continue// by using someone else's crate
 		if(!(obj_flags & EMAGGED) && P.contraband) // will you show me?
 			continue // i'd be right happy to
-		meme_pack_data[P.group]["packs"] += list(list(
+		meme_pack_data[P.group]["packs"] += list(list(//mfw goof sullies your nice cargo features with n a s t y and downright s i l l y comments^^^
 			"name" = P.name,
-			"cost" = P.cost * 2, //displays twice the normal cost
+			"cost" = P.cost,//used to be cost*2
```
Comment (Jordie0608) [inline code/modules/cargo/expressconsole.dm]: Also don't really need this one; git history exists

File: code/modules/cargo/expressconsole.dm [hunk 5]
```diff
@@ -120,7 +120,7 @@
 						CHECK_TICK
 					if(empty_turfs && empty_turfs.len)
 						var/LZ = empty_turfs[rand(empty_turfs.len-1)]
-						SSshuttle.points -= SO.pack.cost * 2
+						SSshuttle.points -= SO.pack.cost * 1.2
```
Comment (Jordie0608) [inline code/modules/cargo/expressconsole.dm]: I believe you'll also want to change this.

**Review by Jordie0608 — CHANGES_REQUESTED** [review]

File: code/modules/cargo/expressconsole.dm (hunk 5 above)
Comment (MrDoomBringer) [reply code/modules/cargo/expressconsole.dm]: i swear i have braincells

**Review by Jordie0608 — DISMISSED** [review]

**Comment by ShizCalev** [comment]
conflict.

**Review by ExcessiveUseOfCobblestone — APPROVED** [review]

**Pull request merged**